modules/apps/27-interchain-accounts/controller/keeper/relay.go

```diff
@@ -15,16 +15,10 @@ import (
 // If the base application has the capability to send on the provided portID. An appropriate
 // absolute timeoutTimestamp must be provided. If the packet is timed out, the channel will be closed.
 // In the case of channel closure, a new channel may be reopened to reconnect to the host chain.
-func (k Keeper) SendTx(ctx sdk.Context, chanCap *capabilitytypes.Capability, portID string, icaPacketData icatypes.InterchainAccountPacketData, timeoutTimestamp uint64) (uint64, error) {
-	// Check for the active channel
-	activeChannelID, found := k.GetActiveChannelID(ctx, portID)
```

Comment:
hmm, shouldn't this check that the channel is in the active channel mapping? Optimistic sends aren't strictly disabled (except via checking active channel mapping)

Reply:
To get the channel capability, an auth module will need to call `GetActiveChannelID(ctx, connectionID, portID)`. Do we still need to enforce the check here? If so I can just pass the connectionID as arg here as opposed to the channelID.

Reply:
Yes I think so, it doesn't hurt to be defensive. What happens if the auth module allows the channel ID to be specified somehow (bypassing GetActiveChannelIID)

Reply:
This is true! I'll update